src/styles/tags.css

```diff
@@ -12,6 +12,10 @@
   justify-content: center;
 }
 
+.tags-list li {
```

Comment:
I could just add the margin to the `tag-pill` class

Reply:
Whoops, nevermind. The `tag-pill` element is inside each `li`, and margin doesn't effect `a` and `span` elements.